specification/migrate/common-types/resource-management/v1/privatelinks.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
can you use the common-types instead of creating another version?

Reply:
the swagger generated from typespec requires the common types to be in a certain directory relative to the current one. I'll check with what changes we took for SDS PR and update.

Reply:
ive removed it. Now there is an error in lint diff. Should i just ignore it?

Reply:
fixed this.